sirius_control/sirius_control/sirius_status.py

```diff
@@ -0,0 +1,78 @@
+import rclpy
+from rclpy.node import Node
+from industrial_msgs.msg import RobotStatus, TriState, RobotMode
+from std_msgs.msg import String
+
+
+class SiriusStatus(Node):
+
+    def __init__(self):
+        super().__init__('sirius_status')
+        self.send_status = self.declare_parameter('send_status', '/sirius_status').value
+        self.default_receive_topics = [
+            '/joy_multiplexer/selected_output',
+            '/relaxing_middleware/state'
+        ]
+        self.receive_topics = self.declare_parameter('receive_topics', self.default_receive_topics).value
```

Comment:
It's not necessary to declare parameters for topic names, as they can be easily remapped - [docs.ros.org](https://docs.ros.org/en/foxy/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Nodes/Understanding-ROS2-Nodes.html#remapping)

Reply:
Resloved in 5355693